xwiki-platform-core/xwiki-platform-appwithinminutes/xwiki-platform-appwithinminutes-ui/src/main/resources/AppWithinMinutes/ClassEditSheet.xml

```diff
@@ -114,13 +114,18 @@ xcontext.put('propertyCustomDisplayer', new PropertyCustomDisplayer(xcontext))
           #set($formFieldDoc = $xwiki.getDocument($formField))
           #set($discard = $formFieldDocs.add($formFieldDoc))
           #set($formFieldIcon = $formFieldDoc.getObject($formFieldClassName).getProperty('icon').value)
-          #if($formFieldIcon.contains('/'))
-            #set($formFieldIconURL = $xwiki.getSkinFile($formFieldIcon))
-          #else
-            #set($formFieldIconURL = $formFieldDoc.getAttachmentURL($formFieldIcon))
+          ## Fallbacks on the default icon set if Silk is not available.
+          #set($formFieldIconRendered = $services.icon.renderHTML($formFieldIcon, 'Silk'))
```

Comment:
Why did you forced it on Silk?

Reply:
Because it was nicer with the other icons. Here is how it looks with Font Awesome:

Reply:
If you really want them aligned, you can give some min-width from CSS.

Reply:
There is actually the css class `fa-fw` that gives a constant width to every icons and align them at the center. The problem is that we probably shouldn't use it for all the icon sets. Maybe giving just a `min-width` can help but how much would it be? Is it going to work nicely with other icon sets?